tests/test_multilabel_classification.py

```diff
@@ -718,3 +722,71 @@ def test_invalid_alpha(self):
         for alpha in [-0.5, 1.5]:
             with pytest.raises(ValueError, match=partial_error_msg):
                 ml_scorer.exponential_moving_average(np.ones(5).reshape(1, -1), alpha=alpha)
+
+
+def flip_labels(label, flip_prob):
+    return 1 - np.array(label) if np.random.rand() < flip_prob else label
+
+
+@composite
+def cleanlab_data_strategy(draw):
+    num_classes = draw(st.integers(min_value=2, max_value=3))
+    num_samples = draw(st.integers(min_value=10, max_value=50))
+
+    # Generate true labels as one-hot encoded vectors for multi-label
+    true_labels = draw(
+        st.lists(
+            st.lists(
+                st.integers(min_value=0, max_value=1),
+                min_size=num_classes,
+                max_size=num_classes,
+            ),
+            min_size=num_samples,
+            max_size=num_samples,
+        )
+    )
+
+    # Generate noise matrix for multi-label
```

Comment:
IIUC, we're flipping an entire one-hot encoded/binarized row this way?

```python
>>> import numpy as np
>>> def flip_labels(label, flip_prob):
...     return 1 - np.array(label) if np.random.rand() < flip_prob else label
... 
>>> true_labels = np.random.randint(0, 2, (6, 4))
>>> true_labels
# array([[0, 0, 0, 1],
#       [1, 1, 0, 1],
#       [0, 0, 0, 0],
#       [1, 1, 0, 0],
#       [1, 0, 0, 1],
#       [1, 0, 0, 1]])
>>> flip_prob = 0.2
>>> noisy_labels = np.array([flip_labels(label, flip_prob) for label in true_labels])
>>> noisy_labels
# array([[0, 0, 0, 1],
#       [1, 1, 0, 1],
#       [0, 0, 0, 0],
#       [1, 1, 0, 0],
#       [0, 1, 1, 0],   # This one is completely flipped
#       [0, 1, 1, 0]])  # This one also.
```

Reply:
It's random now (not across entire row), with the Docstring,